get_latest_metadata.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+if [ -f "/vagrant/.env" ]; then
+  source /vagrant/.env
+fi
+
+# Check out latest version of metadata
+if [ -d $MMD_REPO_FOLDER_NAME ]; then
+  echo "$MMD_REPO_FOLDER_NAME repository exists locally, running git pull."
+  cd $MMD_REPO_FOLDER_NAME
+  # Get commit hash
+  PREV=$(git rev-parse HEAD)
+  git pull || echo "Could not read from git repository - continuing script execution..."
+  NEW=$(git rev-parse HEAD)
+  FILES=$(git diff --name-only $PREV $NEW)
+else
+  echo "Cloning repository."
+  cd $LIB
+  git clone $MMD_XML_REPO || echo "Could not read from git repository - continuing script execution..."
```

Comment:
Should maybe clone into folder you've specified

```bash
git clone $MMD_XML_REPO $MMD_REPO_FOLDER_NAME
```

Reply:
`$MMD_XML_REPO` should have a sane default - pointing to an example repository on Github.